src/python/pants/backend/scala/compile/scalac_test.py

```diff
@@ -584,6 +584,7 @@ def test_compile_with_local_scalac_plugin(
 
                 scala_sources(
                     scalac_plugins=["acyclic"],
+                    dependencies=["3rdparty/jvm:com.lihaoyi_acyclic_2.13"],
```

Comment:
this is a special case where the plugin is also a source code dependency, as the code imports `acyclic.file`